docs/oas/business.yaml

```diff
@@ -2682,6 +2682,142 @@ paths:
+                            postalCode: H0H0H0
+                            addressRegion: BC
+                          officer: 
+                            firstName: Joe
+                            lastName: Swanson
+                            middleName: P
+                            email: joe@email.com
+                            organizationName: 
+                            partyType: person
+                          roles:
+                            - appointmentDate: '2024-07-26'
+                              roleType: Completing Party
+                        - mailingAddress: 
+                            streetAddress: mailing_address - address line one
+                            streetAddressAdditional: 
+                            addressCity: mailing_address city
+                            addressCountry: CA
+                            postalCode: H0H0H0
+                            addressRegion: BC
+                          deliveryAddress: 
+                            streetAddress: mailing_address - address line one
+                            streetAddressAdditional: 
+                            addressCity: mailing_address city
+                            addressCountry: CA
+                            postalCode: H0H0H0
+                            addressRegion: BC
+                          officer: 
+                            firstName: Director
+                            lastName: Swanson
+                            middleName: P
+                            email: joe@email.com
+                            organizationName: 
+                            partyType: person
+                          roles:
+                            - appointmentDate: '2024-07-26'
+                              roleType: Director
+                        shareStructure: 
+                          shareClasses:
+                            - name: Classic 1 Shares
+                              priority: 0
+                              maxNumberOfShares: 0
+                              parValue: 0
+                              currency: string
+                              hasMaximumShares: true
+                              hasParValue: true
+                              hasRightsOrRestrictions: true
+                              series:
+                                - name: string
+                                  priority: 0
+                                  hasMaximumShares: true
+                                  hasRightsOrRestrictions: true
+                      business: 
+                        identifier: TO12345678
+                      header:
+                        accountId: 1234
+                        affectedFilings: []
+                        availableOnPaperOnly: false
+                        certifiedBy: full name
+                        colinIds: []
+                        comments: []
+                        date: 2024-07-25T19:54:02.949741+00:00
```

Comment:
the date format is yyyy-mm-dd 
![image](https://github.com/user-attachments/assets/fe417a1b-c3de-4765-b55f-61de4009b43a)

Reply:
Looks like my morning coffee has not kicked in yet :)